wheelchair_detector/package.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<package>
+  <name>wheelchair_detector</name>
+  <version>0.0.0</version>
```

Comment:
please make sure that this version string is precisely the same as in all other packages in this repo.